app/assets/stylesheets/geoblacklight/modules/icon-customization.scss

```diff
@@ -1,60 +1,50 @@
-// Additional customizations beyond generated FontCustom styles
-
-.geoblacklight-icon {
-  color: $gray-600;
-  font-size: 1.2em;
-}
-
-.geoblacklight-stanford {
-  color: #8c1515;
-}
-
-.geoblacklight-public {
-  @extend .fa, .fa-unlock, .text-muted;
-}
-
-.geoblacklight-restricted {
-  @extend .fa, .fa-lock, .text-muted;
-  width: 17px;
-}
-
-.geoblacklight-citation {
-  @extend .fa, .fa-bookmark;
-}
-
-.geoblacklight-data-dictionary {
-  @extend .fa, .fa-book;
-}
-
-.geoblacklight-sms {
-  @extend .fa, .fa-mobile;
-}
-
-.geoblacklight-email {
-  @extend .fa, .fa-envelope;
-}
-
-.geoblacklight-relations-ancestor {
-  @extend .fa, .fa-pagelines;
-}
-
-.geoblacklight-relations-descendant {
-  @extend .fa, .fa-leaf;
-}
-
-.geoblacklight-web_services {
-  @extend .fa, .fa-external-link;
-}
-
-.geoblacklight-metadata {
-  @extend .fa, .fa-file-text-o;
-}
-
-.geoblacklight-carto {
-  @extend .fa, .fa-map-marker;
-}
-
-// DEPRECATED: Will be removed in GeoBlacklight v2.0
-.geoblacklight-cartodb {
-  @extend .geoblacklight-cartodb;
+// SVG icons via FontAwesome and IcoMoon, see icomoon.json for project data
```

Comment:
Which file is `icomoon.json`?

Reply:
Sorry! Tis the geoblacklight-icons.json file

Reply:
👍